v2/examples/notifications/get_notification_content.go

```diff
@@ -0,0 +1,19 @@
+package examples
+
+import (
+	"context"
+	"log"
+
```

Comment:
any reason to have a line break here?

Reply:
its a Go import linting practice that separates inbuilt language package references vs external deps :)